### PR TITLE
iOS SDK 0.10.0: background uploading of Streemshots & Recordings

### PR DESCRIPTION
**Pull request by dgoldman-pdx**
I could use some Sandbox credentials so that I could do a bit more testing of these builds...

### CODE REVIEW

File: StreemNow/AppDelegate.swift [hunk 1]
```diff
@@ -1,50 +1,25 @@
...
-        NSLog(#function)
-        // Restart any tasks that were paused (or not yet started) while the application was inactive. If the application was previously in the background, optionally refresh the user interface.
-    }
-
-    func applicationWillTerminate(_ application: UIApplication) {
-        NSLog(#function)
-        // Called when the application is about to terminate. Save data if appropriate. See also applicationDidEnterBackground:.
+    
+    func application(_ application: UIApplication, handleEventsForBackgroundURLSession identifier: String, completionHandler: @escaping () -> Void) {
+        Streem.setBackgroundURLSession(withIdentifier: identifier, completionHandler: completionHandler)
```
Comment (seanadkinson) [inline StreemNow/AppDelegate.swift]: Would all clients need to add this line to their `AppDelegate`?  If so, let's add to the README

File: StreemNow/ViewController.swift [hunk 2]
```diff
@@ -71,18 +51,26 @@ class ViewController: UIViewController {
...
-                            self.present(alert, animated: true)
+                            self.presentAlert(message: "Unable to call \(user.name).")
+                            self.isStartingAStreem = false
                             return
                         }
+                        else {
+                            if #available(iOS 11.0, *) {
+                                let session = Streem.sharedInstance.getCurrentARSession()
+                                print("AR Session: \(String(describing: session))")
+                            }
```
Comment (seanadkinson) [inline StreemNow/ViewController.swift]: We can probably get rid of the useless `print`s we have around here

File: StreemNow/AppDelegate.swift (hunk 1 above)
Comment (dgoldman-pdx) [reply StreemNow/AppDelegate.swift]: Good catch!

File: StreemNow/ViewController.swift (hunk 2 above)
Comment (dgoldman-pdx) [reply StreemNow/ViewController.swift]: 👍

**Pull request merged**